docs/ides/web-ides.md

```diff
@@ -144,7 +144,7 @@ command. To add VS Code web as a web IDE, you have two options.
 
    ```hcl
    module "vscode-web" {
-     source         = "https://registry.coder.com/modules/vscode-web"
+     source         = "registry.coder.com/modules/vscode-web/coder"
```

Comment:
```suggestion
     source         = "registry.coder.com/modules/vscode-web/coder"
     version        = "1.0.14"
```
Version is also required when using this format. 
See https://registry.coder.com/modules/vscode-web